components/cam/bld/namelist_files/namelist_definition.xml

```diff
@@ -265,217 +265,52 @@
-       group="nudging_nl" valid_values="" >
-       Day at which Nudging Ends.
-       Default: none
-       </entry>
-
-<entry id="Nudge_Hwin_lo" type="real" category="nudging"
-       group="nudging_nl" valid_values="" >
-       LOW Coeffcient for Horizontal Window.
-       Default: none
-       </entry>
-
-<entry id="Nudge_Hwin_hi" type="real" category="nudging"
-       group="nudging_nl" valid_values="" >
-       HIGH Coeffcient for Horizontal Window.
-       Default: none
-       </entry>
-
-<entry id="Nudge_Hwin_lat0" type="real" category="nudging"
-       group="nudging_nl" valid_values="" >
-       LAT0 of Horizonalt Window.
-       Default: none
-       </entry>
-
-<entry id="Nudge_Hwin_latWidth" type="real" category="nudging"
-       group="nudging_nl" valid_values="" >
-       Width of LAT Window.
-       Default: none
-       </entry>
-
-<entry id="Nudge_Hwin_latDelta" type="real" category="nudging"
-       group="nudging_nl" valid_values="" >
-       Steepness of LAT Window.
-       Default: none
-       </entry>
-
-<entry id="Nudge_Hwin_lon0" type="real" category="nudging"
-       group="nudging_nl" valid_values="" >
-       LON0 of Horizontal Window.
-       Default: none
-       </entry>
-
-<entry id="Nudge_Hwin_lonWidth" type="real" category="nudging"
-       group="nudging_nl" valid_values="" >
-       Width of LON Window.
-       Default: none
-       </entry>
-
-<entry id="Nudge_Hwin_lonDelta" type="real" category="nudging"
-       group="nudging_nl" valid_values="" >
-       Steepness of LON Window.
-       Default: none
-       </entry>
-
-<entry id="Nudge_Vwin_lo" type="real" category="nudging"
-       group="nudging_nl" valid_values="" >
-       LOW Coeffcient for Vertical Window.
-       Default: none
-       </entry>
-
-<entry id="Nudge_Vwin_hi" type="real" category="nudging"
-       group="nudging_nl" valid_values="" >
-       HIGH Coeffcient for Vertical Window.
-       Default: none
-       </entry>
-
-<entry id="Nudge_Vwin_Hindex" type="real" category="nudging"
-       group="nudging_nl" valid_values="" >
-       HIGH Level Index for Verical Window.
-       Default: none
-       </entry>
-
-<entry id="Nudge_Vwin_Hdelta" type="real" category="nudging"
-       group="nudging_nl" valid_values="" >
-       Steepness of HIGH end of Vertical Window.
-       Default: none
-       </entry>
-
-<entry id="Nudge_Vwin_Lindex" type="real" category="nudging"
-       group="nudging_nl" valid_values="" >
-       LOW Level Index for Verical Window.
-       Default: none
+       Linearly interpolate nudging data to current (set to TRUE) or future 
+       (set to FALSE, used by the default model) model time step.
+       Default: FALSE
        </entry>
 
-<entry id="Nudge_Vwin_Ldelta" type="real" category="nudging"
+<entry id="Nudge_File_Ntime" type="integer" category="nudging"
        group="nudging_nl" valid_values="" >
-       Steepness of LOW end of Vertical Window.
-       Default: none
```

Comment:
@jiansunpnnl , it is not clear to me how to specify the nudging time period and nudging spatial window without these options. Are they handled in a different way? Can you clarify, like give an example? Thanks.

Reply:
Hi @tangq , thank for the comments. The nudging time period and nudging spatial window are specified by **the same way** as the default model. The new options **DO NOT** affect these settings. A good example could be found in the Table 3 of Tang et al. 2019.

Reference
Tang, Q., Klein, S. A., Xie, S., Lin, W., Golaz, J.-C., Roesler, E. L., Taylor, M. A., Rasch, P. J., Bader, D. C., Berg, L. K., Caldwell, P., Giangrande, S. E., Neale, R. B., Qian, Y., Riihimaki, L. D., Zender, C. S., Zhang, Y., and Zheng, X.: Regionally refined test bed in E3SM atmosphere model version 1 (EAMv1) and applications for high-resolution modeling, Geosci. Model Dev., 12, 2679–2706, https://doi.org/10.5194/gmd-12-2679-2019, 2019.

Reply:
I think @tangq 's observation is due to a strange behavior of git. Normally, the text shaded in light red would mean to be remove after the PR, which is not true in the current case.

Reply:
@wlin7 , thanks for the clarification. Perhaps, we can report it to github.com. I was confused why these "deleted" options can still be used.

Reply:
Hi @tangq @wlin7 This portion of namelist was duplicated in the original xml file. Jian removed the duplication.

Reply:
Hi @wlin7 and @tangq . Thanks for the clarification. As we have mentioned in the point#4 of this PR (i.e., the bug fixes section), the nudging-related namelist variables are declared twice in the namelist_definition.xml file, which does not seem necessary to me. I deleted the duplicate declarations and the nudged simulation worked fine. This is the reason why you see some variables are deleted in this file. Hope this is clear.

Reply:
Thanks for the clarifications, @kaizhangpnl @jiansunpnnl . It is good to know that it's not a github.com issue.